tests/serialize/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Could you post a link to some documentation and examples of this pattern? It's just that I've never encountered it before nor can I find any documentation in the official `npm` docs or even stackoverflow. I'd just like to avoid taking advantage of an unsupported feature.